libcloudforensics/providers/gcp/internal/compute.py

```diff
@@ -417,6 +417,47 @@ def ListDiskByLabels(self,
     return self._ListByLabel(
         labels_filter, disk_service_object, filter_union)
 
+  def ListReservedExternalIps(self, zone: str) -> List[str]:
+    """Lists all static external IP addresses that are available to a zone.
+
+    The method first converts the zone to a region,
+    and then queries the GCE addresses resource.
+
+    Args:
+      zone (str): The zone in which the returned IPs would be available.
+
+    Returns:
+      List[str]: The list of available IPs in the specified zone.
+
+    Raises:
+      ValueError: If the zone is malformed.
+      errors.ResourceNotFoundError: If the request did not succeed.
+    """
+    # Convert zone to region
+    zone_parts = zone.split('-')
+    if len(zone_parts) != 3:
+      raise ValueError('Invalid zone: {0:s}.'.format(zone))
+    region = '-'.join(zone_parts[:-1])
+    # Request list of addresses
+    addresses_client = self.GceApi().addresses()
+    params = dict(project=self.project_id, region=region)
```

Comment:
```suggestion
    params = {
      'project': self.project_id, 
      'region': region
   }
```
IIRC the python style guide discourages using `dict()` to construct dictionaries.